docs/src/shared/PreviewComponentWithCode.tsx

```diff
@@ -17,11 +17,10 @@ const PreviewComponentWithCode: React.FC<PreviewComponentProps> = (props): JSX.E
             <Box
                 sx={{
                     p: 2,
-                    height: '70vh',
+                    height: { xs: 'calc(70vh - 105px)', sm: 'calc(70vh - 113px)' },
```

Comment:
Do you want to add a comment as to what these specific calculations are doing? Where does the 105 & 113 px come from?

Reply:
Done